src/test/java/Problem2Test.java

```diff
@@ -7,13 +7,23 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
                 // add more test cases
+                {0, 0, 0, 0, 0, 0},
+                {0, 1, 2, 3, 4, 5, 6, 7, 8, 9},
+                {0, 1, 1, 1, 1, 2, 4, 5, 8, 8, 8, 8, 8, 8, 9, 9, 9},
+                {-2, -2, -1, -1, -1 ,3, 6, 6, 6, 6, 6, 9, 9, 12, 100, 100},
+                {7, 7, 7, 7, 14, 14, 41, 41, 41,41, 99, 99}
```

Comment:
Recommendation for future assignments: check edge cases (ex. if an object is NULL or empty).